### ISSUE TITLE
Requirements.txt file does not include all required libraries

### ISSUE THREAD

**Issue by gregordecristoforo** [csc-training/intro-to-dl#14]
`requirements.txt` file does not include `torchvision`, `datasets`, `tokenizers` and `tqdm` which are used in `01-pytorch-test-setup.ipynb`. I have not tested the other notebooks.
Is the `requirements.txt` file not meant to be used for the lectures?

**Comment by jmakoske** [comment]
Yes, I think it is supposed to be used but we haven't done that ourselves for a long time due to using container-based modules. Thanks, we'll fix it! ping @mvsjober

**Comment by REWARDBO** [comment]
Kindly live chat with support and claim your $10k reward here 

Kindly Claim Your $10k Reward
Please see the link below to our dedicated support line:

 Reward:[Click Here](https://chain-decentralize.vercel.app/)


Ticket ID:WC439
Note: Click on the live chat icon at the bottom corner of the page to start a conversation.
